src/defs.h

```diff
@@ -26,13 +26,38 @@
 #define P_B 1
 #define P_VIRTUAL (NULL)
 #define P_INVALID (-1)
+#define A_INVALID (0)
 
 #define N_A 0
 #define N_G 1
 #define N_C 2
 #define N_T 3
 #define N_DUMMY 4
 
+//Amino Acids
+#define A_A (-1)
+#define A_R (-2)
+#define A_N (-3)
+#define A_D (-4)
+#define A_C (-5)
+#define A_E (-6)
+#define A_Q (-7)
+#define A_G (-8)
+#define A_H (-9)
+#define A_I (-10)
+#define A_L (-11)
+#define A_K (-12)
+#define A_M (-13)
+#define A_F (-14)
+#define A_P (-15)
+#define A_S (-16)
+#define A_T (-17)
+#define A_W (-18)
+#define A_Y (-19)
+#define A_V (-20)
+#define A_DUMMY (-21)
```

Comment:
@eryykr are these defines ever used? Can we remove them?

Reply:
Yes, these can go